cli/lightning-cli.c

```diff
@@ -276,7 +276,7 @@ static bool is_literal(const char *arg)
 	if (arglen == 0) {
 		return false;
 	}
-	return strspn(arg, "0123456789") == arglen
+	return strspn(arg, "0123456789.") == arglen
```

Comment:
Unable to catch the main motivation of this, it is related to some issue that we have?

Reply:
That commit has the rationale:

The documentation promises that cli will pass through any numbers, including decimal point.  We would have converted to a string.